pkg/server/server.go

```diff
@@ -7,8 +7,75 @@ import (
 	"path/filepath"
 
 	"github.com/datreeio/admission-webhook-datree/pkg/deploymentConfig"
+	"gopkg.in/yaml.v2"
 )
 
+type ConfigAllowedListsType struct {
+	SkipList     []string `yaml:"skipList"`
+	ValidateList []string `yaml:"validateList"`
+}
+
+var ConfigAllowedLists ConfigAllowedListsType
+
+func InitServerVars() error {
```

Comment:
This is the only method that should be public in this package, isn't it?

Reply:
beside `ValidateCertificate` and `InitServerVars` every other update to private